packages/core/src/plugins/algolia.js

```diff
@@ -24,10 +24,10 @@ function addNoIndexClasses(content) {
     'dropdown',
     'b-modal',
     'panel:not([expanded])',
-    'popover div[slot=content]',
+    'span[data-mb-component-type="popover"] div[data-mb-slot-name="content"]',
     'question div[slot=hint]',
     'question div[slot=answer]',
-    'tab:not(:first-child)',
+    'tab:nth-of-type(n+2)',
     'tab-group:not(:first-child)',
```

Comment:
Should we use `nth-of-type(n+2)` here as well? I don't see problems here currently as we are not introducing new nodes (e.g. `template`) to become the first child, but I'm thinking `nth-of-type(n+2)` may be a more future-proof way to select the node.

Reply:
nice catch here, seems like several of the selectors could use some refining overall
Let's move that to a different PR if you're up for it 🙂

Reply:
Sure! will make a PR for that :-)